vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb

```diff
@@ -25,6 +25,10 @@ def initialize
       @active_tab = "admin_occupancies"
     end
 
+    def dashboard
+      @secure_rooms = current_facility.products(SecureRoom).active
```

Comment:
Hidden means it's hidden from normal users. So like this view: http://localhost:3000/facilities/example